test/Serilog.Sinks.OpenTelemetry.Tests/OpenTelemetryEnvironmentTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Serilog.Helpers;
+using Xunit;
+
+namespace Serilog.Sinks.OpenTelemetry.Tests;
+
+public class OpenTelemetryEnvironmentTests
+{
+    [Fact]
+    public void ConfigureFillOptionsWithEnvironmentVariablesValues()
+    {
+        BatchedOpenTelemetrySinkOptions options = new();
+        var endpoint = "http://localhost";
+        var headers = "header1=1,header2=2";
+        var resourceAttributes = "name1=1,name2=2";
+        Environment.SetEnvironmentVariable("OTEL_EXPORTER_OTLP_ENDPOINT", endpoint);
+        Environment.SetEnvironmentVariable("OTEL_EXPORTER_OTLP_HEADERS", headers);
+        Environment.SetEnvironmentVariable("OTEL_RESOURCE_ATTRIBUTES", resourceAttributes);
```

Comment:
Using the environment variables directly will cause issues if the tests rely on OTLP logging at some point in the future, and will cause tests to interfere with each other when running concurrently.

Perhaps `OpenTelemetryEnvironment` could have an internal `Configure(options, endpointVariable, headersVariable, resourceAttributesVariable)` method added, so that test can call it without setting the actual environment variables?

The "real" configure method could then just be a simple, untested:

```csharp
    public static void Configure(BatchedOpenTelemetrySinkOptions options)
    {
        Configure(
            options,
            Environment.GetEnvironmentVariable("OTEL_EXPORTER_OTLP_ENDPOINT"),
            Environment.SetEnvironmentVariable("OTEL_EXPORTER_OTLP_HEADERS"), ...
```

?

Reply:
Thanks for the advise!

I've fixed it in another way, the Configure method receives the function that can get the env var value, so in the test, I can provide a mock function.